Azrux/Athenas-front
# Install biome

## Description
* Eslint & Prettier were replaced with biome js
* package.json updated

## How to test?
* Install biome extension for vscode
* Run npm i
* You can add this in any component: `let someVar: any;`
* It should throw an error related with no explicit `any`
* Also, you can run this command and check if there's an error: `npx @biomejs/biome lint .`
